manifests/c/CocCoc/CocCoc/120.0.6099.234/CocCoc.CocCoc.locale.vi-VN.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: CocCoc.CocCoc
+PackageVersion: 120.0.6099.234
+PackageLocale: vi-VN
+Publisher: Coc Coc Co., Ltd.

```

```suggestion
Publisher: Công ty TNHH Cốc Cốc
```
This should match what is written to control panel